sklearn/covariance/_graph_lasso.py

```diff
@@ -345,7 +345,7 @@ class BaseGraphicalLasso(EmpiricalCovariance):
         "tol": [Interval(Real, 0, None, closed="right")],
         "enet_tol": [Interval(Real, 0, None, closed="right")],
         "max_iter": [Interval(Integral, 0, None, closed="left")],
-        "mode": [StrOptions({"cd", "lars"})],
+        "mode": [StrOptions({"cd", "lars"})],  # type: ignore
```

Comment:
I searched for a way to configure mypy so that we don't have to add this comment, but yet have not found any.

Reply:
Maybe [this](https://github.com/scikit-learn/scikit-learn/pull/24137#discussion_r957577658) will work?

Reply:
Indeed, thanks for the link!